wft4galaxy/common.py

```diff
@@ -12,17 +12,21 @@
 ENV_KEY_GALAXY_URL = "GALAXY_URL"
 ENV_KEY_GALAXY_API_KEY = "GALAXY_API_KEY"
 
+# configure logger
+_log_format = "%(asctime)s [wft4galaxy] [%(levelname)-5.5s]  %(message)s"
+default_logger = _logging.getLogger("WorkflowTest")
```

Comment:
I would also wrap these lines (17-21) into a `config_logging()` function that gets called when the application is started, not when the module is loaded.